index.js

```diff
@@ -1,60 +1,67 @@
-/* eslint-disable no-use-before-define */
-const booksContainer = document.querySelector('.books');
-const titleInput = document.getElementById('title-input');
-const authorInput = document.getElementById('author-input');
-
-const addBtn = document.querySelector('.add-btn');
-
-let Books = JSON.parse(localStorage.getItem('books')) || [];
-
-addBtn.addEventListener('click', () => {
-  const titleValue = titleInput.value;
-  const authorValue = authorInput.value;
-  const bookID = Date.now();
-  addBook(titleValue, authorValue, bookID);
-});
-
-displayBooks();
-
-function addBook(title, author, id) {
-  const book = {
-    title,
-    author,
-    ID: id,
-  };
-  Books.push(book);
-  localStorage.setItem('books', JSON.stringify(Books));
-  displayBooks();
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
 }
+class Library {
+  constructor(container, titleInput, authorInput, addBtn) {
+    this.booksContainer = document.querySelector(container);
+    this.titleInput = document.getElementById(titleInput);
+    this.authorInput = document.getElementById(authorInput);
+    this.addBtn = document.querySelector(addBtn);
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+    this.displayBooks();
+    this.addEventListeners();
+  }
 
-function removeBook(id) {
-  Books = Books.filter((book) => book.ID !== id);
-  localStorage.setItem('books', JSON.stringify(Books));
-  displayBooks();
-}
-
-function displayBooks() {
-  booksContainer.innerHTML = ''; // Clear the books container before rendering new books
-  Books.forEach((book) => {
-    const div = document.createElement('div');
-    div.id = book.ID;
+  addBook(title, author, id) {
+    const book = new Book(title, author, id);
+    this.books.push(book);
```

Comment:
- [ ] Optional:
- I would suggest you use the array methods that doesn't mutate the array, e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push)